modules/irsa/outputs.tf

```diff
@@ -12,3 +12,14 @@ output "external_dns" {
     external_dns_use_eks_idp = var.use_cluster_odc_idp
   } : null
 }
+
+output "flyte" {
+  description = "Flyte info"
+  value = var.flyte.enabled ? {
+    eks = {
+      account_id            = regex("arn:.+:.+::(.*):.*", local.oidc_provider_arn)[0]
```

Comment:
Nitpick
This seems a bit fragile, given that the account id can be the cluster or the one created here, we should pass in the the account id from the eks module, and use `data "aws_caller_identity" "current" {}` to the the id from the current session, use the similar logic to `local.oidc_provider_arn` to determine the value of a new variable `oidc_account`